internal/grpc/services/authprovider/authprovider.go

```diff
@@ -149,9 +149,9 @@ func (s *service) Authenticate(ctx context.Context, req *provider.AuthenticateRe
 			Status: status.NewPermissionDenied(ctx, v, "wrong password"),
 		}, nil
 	case errtypes.NotFound:
-		log.Debug().Str("client_id", username).Msg("unknown client id")
+		log.Debug().Str("username", username).Msg("unknown user")
```

Comment:
```suggestion
		log.Debug().Str("client_id", username).Err(err).Msg("unknown  client id or user")
```

Reply:
What I had in mind after reading your initial comment. :+1:

Reply:
(There is a double white space in the error message)